templates/user/settings/_options.html.twig

```diff
@@ -25,6 +25,12 @@
                 {{ 'password_and_2fa'|trans }}
             </a>
         </li>
+        <li>
```

Comment:
might be worth not showing this if user is admin, and maybe any backend checks people think are necessary to make sure the admin doesn't immediately delete themselves

Reply:
Yeah you're absolutely right

Reply:
@melroy89 @nobodyatroot @asdfzdfj what do you think about this? Do we need to disable the form for admins? I mean it is only possible to make people admins via cli, and we can always create new admins via cli. I mean most admins would never use it anyway, so might as well disable it, but I am not sure...

Reply:
Can we also delete an admin via CLI (if case the admin really wants to)? Then yes, you do want to not show this UI option I believe. Since this could often only end up in a bad day.

Reply:
I don't think we have a command for deleting an admin, yet...

Reply:
Now we have. Thanks!